src/organisation/containers/organisation/organisation.component.html

```diff
@@ -3,16 +3,49 @@
     <tbody class="govuk-table__body">
     <tr class="govuk-table__row">
       <th class="govuk-table__header" scope="row">Name</th>
-      <td class="govuk-table__cell">{{orgData[0]}}</td>
+      <td class="govuk-table__cell">{{orgData[0].name}}</td>
       <td class="govuk-table__cell"></td>
     </tr>
     <tr class="govuk-table__row">
       <th class="govuk-table__header" scope="row">Address</th>
       <td class="govuk-table__cell">
-        <span *ngFor="let addressLine of orgData | slice:1">{{addressLine}}<br></span>
+        <div>{{orgData[0].contactInformation.addressLine1}}</div>
+        <div>{{orgData[0].contactInformation.addressLine2}}</div>
```

Comment:
why orgData is an array? could this be the *ngFor

Reply:
orgData is an array but the first object orgData[0] is an object